loomio/loomio
# remove avatar setting js so we can switch off lazy precompile

**CR**: :white_check_mark: 

**QA**: :white_check_mark:

> :white_check_mark: pull down locally, check app loads
> 
> _test core functionality of feature:_
> :white_check_mark: changed my avatar to each type
> 
> _test functionality adjacent to this feature:_
> :white_check_mark: saw my new avatar in discussion
> 
> :ok:  public/ private functionality  **NA**
> :ok: logged out functionality **NA**
> 
> :ok: test in IE 8,9 **NA**
> :question: test in production environment - not sure if this is relevant ... do we need to change anything?

---

**CO signoff**: :question:
**DO signoff**: :ok: took AJAX out, but pretty insignificant UI change

**PO signoff**: :question:

---

Brought to you by everyone's favourite pattern: removing error-prone js and using a redirect instead.
